reference/orders.v3.yml

```diff
@@ -2588,6 +2588,8 @@ components:
         * `HANDLING`
         * `TAX`
         * `FEE`
```

Comment:
while you are at it, may you pls help to update the list of "quantity bound item" to only these:
* `PRODUCT`
* `GIFT_WRAPPING`